composer.json

```diff
@@ -128,6 +124,11 @@
     "extra": {
         "symfony": {
             "allow-contrib": true
+        },
+        "patches": {
+            "knplabs/doctrine-behaviors": {
+                "Old Configuration": "patches/doctrine-behaviors-configuration.patch"
```

Comment:
This needs to be patched, because there is no release https://github.com/KnpLabs/DoctrineBehaviors/pull/432